app/scripts/modules/core/src/cluster/rollups.less

```diff
@@ -182,20 +182,27 @@ table.instances {
 
 .rollup-title-cell {
   flex: 1;
+  min-width: 0;
   display: flex;
   height: 36px;
   line-height: 36px;
   font-size: 16px;
   color: var(--color-primary);
   .pod-center {
     flex: 1 1 auto;
+    min-width: 0;
     display: flex;
     justify-content: space-between;
     background-color: var(--color-accent-g2);
     padding: 0 10px;
     border-right: 1px solid var(--color-alabaster);
     border-left: 1px solid var(--color-alabaster);
     font-weight: 600;
+    > div {
+      white-space: nowrap;
+      overflow-x: hidden;
+      text-overflow: ellipsis;;
```

Comment:
;;

Reply:
Ok I’ll add the third semicolon;;;

Reply:
Ok I’ll add the third semicolon;;;

Reply:
He's going to add a third semicolon apparently

Reply:
;;;